daq/test_modules/ipaddr_module.py

```diff
@@ -102,6 +103,15 @@ def _ip_change_test(self):
         self.host.gateway.request_new_ip(self.host.target_mac)
         self._ip_callback = self._next_test
 
+    def _dhcp_change_test(self):
+        self._set_timeout()
+        if not self.host.connect_port(False):
+            self._logger.error('disconnect port not enabled')
```

Comment:
I thought for this test you shouldn't disconnect port, so that it can test if the device can accept a new IP from the dhcp server.

Reply:
Nevermind my last comment. I thought it was ip_change test.